src/services/fileServices/MdPageServices/__tests__/MediaFileService.spec.js

```diff
@@ -236,4 +237,40 @@ describe("Media File Service", () => {
       )
     })
   })
+
+  describe("DeleteMultipleFiles", () => {
+    it("rejects page names with special characters", async () => {
+      await expect(
+        service.deleteMultipleFiles(sessionData, mockGithubSessionData, {
+          items: [
+            { filePath: "file/file%%%name.pdf", sha },
```

Comment:
was the point of this to check for encoding issues hahah

Reply:
Oops not really haha, it was just a randomly chosen special character